articles/native-platforms/android/index.yml

```diff
@@ -15,3 +15,8 @@ snippets:
 seo_alias: android
 articles:
   - 01-login
+  - 02-custom-login
+  - 03-session-management
```

Comment:
@tenmaster Throws build error, the filename is `03-session-handling` https://github.com/auth0/docs/pull/1638/files

Also, until now I haven't seen PRs for `04-user-profile` and `05-linking-accounts`, so these entries throw build errors as well. I removed the entries locally in order to build. This cannot be merged until `04-user-profile` and `05-linking-accounts` are also merged. 

It would be good in the feature to group this kind of PRs together and deliver them as a bundle.